rdbms/src/main/java/io/spine/server/storage/jdbc/JdbcStorageFactory.java

```diff
@@ -0,0 +1,219 @@
+        return storage;
+    }
+
+    /**
+     * Closes used {@link DataSourceWrapper}.
+     */
+    @Override
+    public void close() {
+        dataSource.close();
+    }
+
+    public static Builder newBuilder() {
+        return new Builder();
+    }
+
+    /**
+     * Builds instances of {@code JdbcStorageFactory}.
+     */
+    public static class Builder {
+
+        private DataSourceWrapper dataSource;
+        private boolean multitenant;
+        private ColumnTypeRegistry<? extends JdbcColumnType<? super Object, ? super Object>> columnTypeRegistry;
+
+        private Builder() {
+        }
+
+        /**
+         * Sets the {@link ColumnTypeRegistry} to use in the generated storages.
+         *
+         * <p>The default value is
+         * {@link JdbcTypeRegistryFactory#defaultInstance() JdbcTypeRegistryFactory.defaultInstance()}.
+         *
+         * <p>To reuse the existent {@linkplain JdbcColumnType column types}, use
+         * {@link JdbcTypeRegistryFactory#predefinedValuesAnd() JdbcTypeRegistryFactory.predefinedValuesAnd()}.
+         *
+         * @param columnTypeRegistry the custom {@link ColumnTypeRegistry} to use in the generated
+         *                           storages
+         */
+        public Builder setColumnTypeRegistry(
+                ColumnTypeRegistry<? extends JdbcColumnType<? super Object, ? super Object>>
+                        columnTypeRegistry) {
+            this.columnTypeRegistry = columnTypeRegistry;
+            return this;
+        }
+
+        /**
+         * Sets optional field {@code isMultitenant}. {@code false} is used by default.
+         */
+        public Builder setMultitenant(boolean multitenant) {
+            this.multitenant = multitenant;
+            return this;
+        }
+
+        /**
+         * Sets required field {@code dataSource}.
+         */
+        public Builder setDataSource(DataSourceWrapper dataSource) {
+            this.dataSource = dataSource;
+            return this;
+        }
+
+        /**
+         * Sets required field {@code dataSource} from wrapped {@link DataSource}.
+         *
+         * @see DataSourceWrapper#wrap(DataSource)
+         */
+        public Builder setDataSource(DataSource dataSource) {
+            this.dataSource = DataSourceWrapper.wrap(dataSource);
+            return this;
+        }
+
+        /**
+         * Sets required field {@code dataSource} from {@link DataSourceConfig}.
+         *
+         * @see HikariConfig
+         * @see DefaultDataSourceConfigConverter#convert(DataSourceConfig)
+         */
+        public Builder setDataSource(DataSourceConfig dataSource) {
+            final HikariConfig hikariConfig = DefaultDataSourceConfigConverter.convert(dataSource);
+            this.dataSource = DataSourceWrapper.wrap(new HikariDataSource(hikariConfig));
+            return this;
+        }
+
+        /**
+         * @return New instance of {@code JdbcStorageFactory}.
```

Comment:
`New` should be `new`. Please check the rest of the cases and address the similar issues.